packages/terra-table/src/TableSubheader.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { PropTypes } from 'react';
```

Comment:
In the terra-table package.json, add prop-types as a dependency.
https://github.com/cerner/terra-core/blob/master/packages/terra-badge/package.json#L51